packages/react/src/SubdomainNavBar/SubdomainNavBar.module.css

```diff
@@ -609,6 +613,18 @@
   line-height: var(--brand-text-lineHeight-400);
 }
 
+.SubdomainNavBar-search-result-item-suggestion {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  white-space: nowrap;
+  border-width: 0;
+}
+
```

Comment:
Think this can be removed as we have a `visually-hidden` global style ([link](https://github.com/primer/brand/blob/ce969048bd332a00f3f8d70deba15b44edcc4fd8/packages/react/src/css/utilities.css#L6))